### ISSUE TITLE
feat: A high-contrast dark theme

### ISSUE THREAD

**Issue by HubKing** [GradienceTeam/Community#65]
### Feature description

Dark theme is common these days, but one problem of dark themes is that they often lack contrast. That is, they use light grey text on dark grey background. Or, when they use pure-black background, they lack borders, so UI elements are not discernable. For some users, this lack of contrast is not just a matter of aesthetics, but of accessibility.

Some applications, like JetBrain's IDE or VS Code, have high-contrast dark theme like below. The background is black and the text is white, so they are easy to read, and the UI elements have white borderlines, so they don't look all mingled together.

It seems there is no high-contrast dark theme in the presets, and I failed to make such a theme myself. How about adding a high-contrast dark theme in the presets?

### Additional Context
JetBrains' IDE

![image](https://blog.jetbrains.com/wp-content/uploads/2018/10/idea-HighContrastTheme.png)

VS Code

![image](https://cdn.hashnode.com/res/hashnode/image/upload/v1520709177817/ByfrAjbFf.png)

MS Windows 10

![image](https://www.maxability.co.in/wp-content/uploads/2021/06/High-contrast-home-page-windows-10.png)

### Did you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/GradienceTeam/Gradience/blob/main/CODE_OF_CONDUCT.md).

**Comment by tfuxu** [comment]
Libadwaita (and normal Gtk) applications already have high contrast versions, which you can enable in Accessibility page in GNOME Control Center.
![Screenshot from 2022-11-12 12-54-14](https://user-images.githubusercontent.com/73042332/201472771-25dfdade-4f23-4d19-99c4-54fd53e403f1.png)

**Comment by HubKing** [comment]
Well, it could work for some people, but that is not exactly what I wanted. I am not visually-impaired, I just want black background + white general text, darker background + brighter-coloured text for special elements like buttons, and white borders. If I enable that accessibility feature, it would look like below. The icons look ugly, the background is still grey, not black, and the text is still dim, not white. It also, for some reason, makes some applications use light theme, not dark.

![image](https://user-images.githubusercontent.com/10900489/201505671-484de057-638f-4750-8090-3f47d267e7cd.png)

**Comment by tfuxu** [comment]
Well, if you want it ASAP, you can create it with Gradience and create a PR here with your theme file, so it could be added to our repository for others to download. You can also tell us here how this theme should look exactly, and wait until someone from our team or some other interested contributor will create a PR for it.

**Comment by HubKing** [comment]
@tfuxu  No, I do not need it urgently. I had tried to create one, but there were so many entries, so I did not know what exactly each of them were for and could not make a good-looking one. If I changed one thing and then another thing looked bad, and if I fixed that thing, then the previous one looked bad, etc.

**Comment by tfuxu** [comment]
> No, I do not need it urgently.

Alright then, we will create it at some point and notify you when PR will be available, so you could review it.

**Comment by tfuxu** [comment]
> I had tried to create one, but there were so many entries, so I did not know what exactly each of them were for and could not make a good-looking one.

Hmm, we should look into that. Maybe a visual representation (more than just Gradience window changing appearance) could help to understand some color sections.

**Comment by daudix-UFO** [comment]
What about something like this?

[high-contrast.zip](https://github.com/GradienceTeam/Community/files/10087410/high-contrast.zip)

**Comment by HubKing** [comment]
> What about something like this?
> 
> [high-contrast.zip](https://github.com/GradienceTeam/Community/files/10087410/high-contrast.zip)

How can I test it? I don't know how to import a ZIP file.

**Comment by daudix-UFO** [comment]
Unzip it and import .json

**Comment by HubKing** [comment]
> Unzip it and import .json

I figured it out, so I was trying to post delete that comment but you have already seen it.

Any way, the high-contrast theme looks good, but I have one question. Does Gnome theme allow customising window borders? I am not sure if it does, but if it does, I think a white window border would be helpful, because currently, when two windows are overlapping, I cannot easily distinguish them.

![image](https://user-images.githubusercontent.com/10900489/204105404-5a767a26-d962-4b7d-a4c2-32eb2df79014.png)

**Comment by daudix-UFO** [comment]
I think there is option to do so ;)

**Comment by daudix-UFO** [comment]
Figured out how to add outlune, looks pretty good

![изображение](https://user-images.githubusercontent.com/77155297/204841166-289888e9-5f36-4e0d-968a-ee0d2cf0f8a8.png)

**Comment by tfuxu** [comment]
Daudix needed to override some CSS rules using Custom CSS editor in Advanced tab to customize window borders, because GTK doesn't expose it directly as `@define-color` variables. You don't need a third-party tool to do this, but you need some knowledge of CSS to customize those more hidden from customization elements in GTK.
